external/netutils/netlib_netmon.c

```diff
@@ -0,0 +1,188 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#if defined(CONFIG_NET) && (CONFIG_NSOCKET_DESCRIPTORS > 0)
```

Comment:
if CONFIG_NET was enabled then Makefile wouldn't build the netlib_netmon.c. So we can remove CONFIG_NET here.

Reply:
In case CONFIG_NET is not defined, netlib_netmon.c will not be compiled. Without CONFIG_NET setting, I think network monitor is meaningless. Please let me know your opinion.

Reply:
I mis-type above the comment, if CONFIG_NET wasn't enabled then Makefile wouldn't build it. so I thought CONFIG_NET is not necessary. but it's trivial fix. so you can ignore it.